### DISCUSSION TITLE
How to change compress bitwindow for zlib?

### DISCUSSION THREAD

**Discussion by jozhn** [icsharpcode/SharpZipLib#771] [Q&A]
I did not find the way to control compress options but only compress level. But  I need to change the wbits of zlib compress.

**Comment by piksel** [comment]
What do you mean by "zlib"? SharpZipLib does not use `zlib`. The `DeflaterEngine` uses a constant window:
https://github.com/icsharpcode/SharpZipLib/blob/519ed7367daf0a5ad7f2d36bee6e994ffcd7e9e8/src/ICSharpCode.SharpZipLib/Zip/Compression/DeflaterEngine.cs#L87

https://github.com/icsharpcode/SharpZipLib/blob/519ed7367daf0a5ad7f2d36bee6e994ffcd7e9e8/src/ICSharpCode.SharpZipLib/Zip/Compression/DeflaterConstants.cs#L51-L59

Not sure if another size could even be used without breaking zip spec...
